workspace_test.go

```diff
@@ -622,3 +622,113 @@ func TestNewWorkspaceConfig(t *testing.T) {
 		assert.Equal(t, output.Valid, true, output.Diagnostics)
 	})
 }
+
+func TestWillDestroy(t *testing.T) {
```

Comment:
Missing coverage for a deletion of a non-matching resource type

Reply:
mm good point